test/jasmine/tests/waterfall_test.js

```diff
@@ -1424,7 +1486,7 @@ describe('waterfall hover', function() {
             })
             .then(function() {
                 assertHoverLabelContent({
-                    nums: '2.2\n4.4 ▲\nInitial: −2.2',
+                    nums: '2.2\n2.2\n4.4 ▲\nInitial: −2.2',
```

Comment:
Why do we have an extra `2.2` now?

Reply:
In respect to https://github.com/plotly/plotly.js/pull/3963#discussion_r293856811 displaying delta seems to be useful especially when there is a base; also when hovering using `hoverinfo`: `delta`.

Reply:
The extra 2.2 here is the `final` value which is the same as `y`.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/3963/commits/0a588366201a20d2d90483d19e42f89e06555126.